boards/arm/sama5/giant-board/configs/nsh/defconfig

```diff
@@ -0,0 +1,80 @@
+#
+# This file is autogenerated: PLEASE DO NOT EDIT IT.
+#
+# You can use "make menuconfig" to make any modifications to the installed .config file.
+# You can then do "make savedefconfig" to generate a new defconfig file that includes your
+# modifications.
+#
+# CONFIG_AUDIO_FORMAT_MP3 is not set
+# CONFIG_NSH_CMDOPT_HEXDUMP is not set
+# CONFIG_SAMA5_UART0 is not set
+# CONFIG_SAMA5_UHPHS_RHPORT1 is not set
+CONFIG_ARCH="arm"
+CONFIG_ARCH_BOARD="sama5d2-xult"
+CONFIG_ARCH_BOARD_GIANT_BOARD=y
+CONFIG_ARCH_BUTTONS=y
```

Comment:
See comments about.  Please refresh this configuration.  Remove button selections.

If the configuration is not tested, please remove it from the PR

Reply:
@patacongo will fix